.env.example

```diff
@@ -62,3 +62,6 @@ ADALITE_ENABLE_LEDGER="true"
 
 # delegation by ticker
 ADALITE_ENABLE_SEARCH_BY_TICKER=true 
+
+# catalyst next round start and end dates in UNIX format (example: 2021-2030)
+ADALITE_NEXT_VOTING_UNIX_PAIR=1609459200000, 1893456000000
```

Comment:
I'd rather use ISO timestamps which are easier to sanity-check on first sight, and split the start and end into separate env variables

Reply:
also, don't forget to add this environment variable to the preview app env variables in heroku, otherwise the deployment fails. I added it, but another commit to trigger the deploy is probably needed

Reply:
I agree with the ISO format. However, the two timestamps don't make much sense without each other for me. This way, the person who sets them will not accidentally forget to set the other one, because they're in pair. When they're in this natural order, the mistake of setting them contrariwise is mitigated to some extent as well, we can even .sort() them in the worst case when they're like this

Reply:
the presence of both timestamps (or neither) and order can be enforced when validating the config. I would rather not be too graceful with their handling - anything odd would be intercepted when deploying as that's when we validate the config as well (if I'm not mistaken)

Reply:
fixed